modules/python/init.zsh

```diff
@@ -44,7 +44,7 @@ fi
 unset local_pyenv
 
 # Return if requirements are not found.
-if (( ! $#commands[(i)python[23]#] && ! $+functions[pyenv] && ! $+commands[conda] )); then
+if (( ! $+commands[(i)python[23]] && ! $+functions[pyenv] && ! $+commands[conda] )); then
```

Comment:
This is the broken line... there are several ways to fix: https://github.com/sorin-ionescu/prezto/issues/1949#issuecomment-891561833